ember-app/app/components/columns/hb-column.js

```diff
@@ -113,7 +113,7 @@ var HbColumnComponent = Ember.Component.extend(SortableMixin, ScrollingColumn, {
 
     var issuesLength = this.get('visibleIssues').length;
     if(horizonVisible && issuesLength < totalColumnLength){
-      Ember.run.debounce(this, 'revealIssues' , horizon, 250);
+      Ember.run.debounce(this, 'revealIssues' , horizon, 200);
```

Comment:
I'd love to see what this feels like as a throttle

I think throttle is more appropriate in this situation, because you want the event to fire right away just not too many calls in a row, where as debounce as to wait until you've scrolled all the way to the end and then it pauses if you scroll really fast.